handler/src/main/java/io/netty/handler/ssl/SslProvider.java

```diff
@@ -35,5 +35,21 @@
      * OpenSSL-based implementation which does not have finalizers and instead implements {@link ReferenceCounted}.
      */
     @UnstableApi
-    OPENSSL_REFCNT
+    OPENSSL_REFCNT;
+
+    /**
+     * Returns {@code true} if the specified {@link SslProvider} supports
+     * <a href="https://tools.ietf.org/html/rfc7301#section-6">TLS ALPN Extension</a>, {@code false} otherwise.
+     */
+    public static boolean isAlpnSupported(final SslProvider provider) {
```

Comment:
@idelpivnitskiy I wonder if we even should consider making it a non static method and just part of the enum itself ?

Reply:
I don't have a strong preference here. If we want to move `SslContext.defaultClientProvider()` and `SslProvider.defaultServerProvider()` here, it would be nice to be consistent and use `static` method in both cases.
If you worry that someone may forget to add a new case to the `switch` when a new value will be added, I can write a test to verify that all values do not throw.

Reply:
I guess we can just keep it as it is for now... :)